homeassistant/components/media_player/__init__.py

```diff
@@ -47,10 +47,6 @@
     SERVICE_VOLUME_MUTE,
     SERVICE_VOLUME_SET,
     SERVICE_VOLUME_UP,
-    STATE_IDLE,
-    STATE_OFF,
-    STATE_PLAYING,
-    STATE_STANDBY,
```

Comment:
Removing these seems to cause test errors. They should have been imported from `homeassistant.const` anyway, so I think fixing the tests will be enough and they can be removed here.

Reply:
We should not remove imports during the deprecation period, I think. It's ok to do after.

Reply:
I have reverted the change to the import, but I've kept the changes to the tests.